lib/internal/validators.js

```diff
@@ -220,7 +220,7 @@ function validateBoolean(value, name) {
 }
 
 /**
- * @param {?object} options
+ * @param {object} [options]
```

Comment:
The parameter is not optional, it might be nullish, but it's not at all optional. We could use `any` instead to clarify that value would be valid.
```suggestion
 * @param {any} options
```

Reply:
I believe this is not resolved. Unresolving.

Reply:
My bad. Silly Github showed me the old diff.